core/block/import/source/source.go

```diff
@@ -37,3 +38,13 @@ func isSupportedExtension(ext string, expectedExt []string) bool {
 func isFileAllowedToImport(fileName, ext string, expectedExt, includeFiles []string) bool {
 	return isSupportedExtension(ext, expectedExt) || lo.Contains(includeFiles, filepath.Base(fileName))
 }
+
+func CountFilesWithGivenExtension(fileReaders map[string]io.ReadCloser, extension string) int {
```

Comment:
We use this function to calculate count of files with extensions like html or md. Because source directory or zip archive can contain not only html or md files, but also image files or video.